### DISCUSSION TITLE
Latest "add multi arch docker build" Error on :edge-arm

### DISCUSSION THREAD

**Discussion by quella** [FreshRSS/FreshRSS#5821] [Q&A] [answered]
Just a quick question and I did see the recent post and details involving changes to the docker image builds.  It seems when I try to do a docker update on FreshRSS using "edge-arm" as my version, I now get an error.  It says , "ERROR: manifest for freshrss/freshrss:edge-arm not found: manifest unknown: manifest unknown"  Based on the recent change to the repository and multi builds, do I just need to now revert to ":edge" and not specify the "-arm" portions.  Maybe I'm to quick to updating and the ":edge-arm" builds are not yet on the hub.  Thanks for all your great work on this wonderful tool.

If it is helpful, I'm running the latest edge-arm docker image on a raspberry Pi for many years.

**Comment by Alkarex** [comment, marked as answer]
Please try with `freshrss/freshrss:edge` without the `-arm` suffix

**Reply by quella** [reply]
That worked to remove the '-arm' from the docker-compose.yaml.  Thanks @Alkarex for confirming this change.  All running well.
